scene/resources/surface_tool.cpp

```diff
@@ -882,7 +882,7 @@ void SurfaceTool::create_vertex_array_from_triangle_arrays(const Array &p_arrays
 		if (lformat & RS::ARRAY_FORMAT_TANGENT) {
 			Plane p(tarr[i * 4 + 0], tarr[i * 4 + 1], tarr[i * 4 + 2], tarr[i * 4 + 3]);
```

Comment:
This is a Vector4. It is not a geometric plane.

Reply:
I removed the Plane entirely